searchcore/src/vespa/searchcore/proton/server/storeonlyfeedview.cpp

```diff
@@ -819,10 +817,12 @@ StoreOnlyFeedView::handleCompactLidSpace(const CompactLidSpaceOperation &op)
                                                                 _gidToLidChangeHandler.grab_pending_changes(),
                                                                 DoneCallback()));
         commitContext->holdUnblockShrinkLidSpace();
-        internalForceCommit(serialNum, commitContext);
+        internalForceCommit(CommitParam(serialNum, true), commitContext);
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour.

Reply:
Removed semantic change